cypress.json

```diff
@@ -7,6 +7,7 @@
     "videosFolder": "tests/cypress/videos",
     "supportFile": "tests/cypress/support",
     "projectId": "tt5hbz",
+    "defaultCommandTimeout": 10000,
```

Comment:
I also was testing this on my branch. I don't think it is addressing the flakes unfortunately but its good to know about